common/src/main/java/com/microsoft/identity/common/internal/cache/ADALOAuth2TokenCache.java

```diff
@@ -134,7 +135,7 @@ public void saveTokens(
         Logger.info(TAG + ":" + methodName, "Setting items to cache for user...");
         for (final String cacheIdentifier : account.getCacheIdentifiers()) {
             //Azure AD Uses Resource and Not Scope... but we didn't override... heads up
-            final String scope = request.getScope();
+            final String scope = StringUtil.convertSetToString(request.getScope(), " ");
```

Comment:
Did we change scope string in request to a set?  If so, why?  The intention of the Authorization request is to be as close to the wire as possible.  It's a space delimited string in the spec... not a set.